### PR TITLE
an/SKY-66: SKY URL Path Post added back

### PR DESCRIPTION
**Pull request by anurmamedov**
SKY URL Path Post added back

### CODE REVIEW

**Review by anurmamedov — COMMENTED** [review]
please check changes

**Review by anurmamedov — COMMENTED** [review]
redirect view added on main page into posts.

**Review by anurmamedov — COMMENTED** [review]
added RedirectView, from main page will be directed to post.urls

**Review by anurmamedov — COMMENTED** [review]
added RedirectView, from main page will be directed to post.urls

**Review by anurmamedov — COMMENTED** [review]
Posts.url added back and loginrequired added for redirecting to login page from posts

File: posts/views.py [hunk 1]
```diff
@@ -5,19 +5,22 @@
...
+    login_url = reverse_lazy('login')
 
     def get_queryset(self):
         base_query = super().get_queryset()
         data = base_query.order_by('-created_date', '-created_time')
         return data
-     
+    
+      
 class PostEditView(UpdateView):
```
Comment (Aymammet) [inline posts/views.py:24]: Why we didn't put this "LoginRequiredMixin" here?

**Review by Aymammet — CHANGES_REQUESTED** [review]

**Review by Aymammet — CHANGES_REQUESTED** [review]

File: posts/views.py (hunk 1 above)
Comment (anurmamedov) [reply posts/views.py:24]: i dont know, do you want me to open ticket and inlude it in all of them ?

**Comment by anurmamedov** [comment]
LoginRequiredMixin added in PostEditView, PostCreateView and PostDeleteView

File: posts/views.py (hunk 1 above)
Comment (anurmamedov) [reply posts/views.py]: added in all of them

**Review by anurmamedov — COMMENTED** [review]
LoginRequiredMixin added in PostEditView, PostCreateView and PostDeleteView

**Review by Aymammet — APPROVED** [review]

**Pull request merged**